BetterSnippetManager.py

```diff
@@ -59,8 +59,6 @@ def run(self):
         snippets_folder = get_settings().get('snippets_folder')
         extension = get_snippet_extension()
         self.SNIPPETS_PATH = os.path.join(sublime.packages_path(), 'User')
-        if snippets_folder:
-            self.SNIPPETS_PATH = os.path.join(self.SNIPPETS_PATH, snippets_folder)
 
```

Comment:
These two lines caused an error when snippets_folder setting was set to a string, and you couldn't edit snippets anymore. Without them, it scans snippets starting from User folder, and all subfolders, and that's better if your snippets aren't saved in User/Snippets/....